<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>_YC_</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Orijinal Resim</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>441</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dönüştürülen Resim</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>120</y>
      <width>441</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDosya">
    <property name="title">
     <string>Dosya</string>
    </property>
    <addaction name="actionResim_A"/>
   </widget>
   <widget class="QMenu" name="menu_1">
    <property name="title">
     <string>_1_</string>
    </property>
    <widget class="QMenu" name="menurgb_2_gray">
     <property name="title">
      <string>rgb_2_gray</string>
     </property>
     <addaction name="actionPython"/>
     <addaction name="actionOpenCv"/>
    </widget>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>histogram</string>
     </property>
     <addaction name="actionPython_2"/>
     <addaction name="actionOpenCv_2"/>
    </widget>
    <widget class="QMenu" name="menuesikleme">
     <property name="title">
      <string>esikleme</string>
     </property>
     <addaction name="actionPython_3"/>
     <addaction name="actionOpenCv_3"/>
    </widget>
    <widget class="QMenu" name="menunegatif">
     <property name="title">
      <string>negatif</string>
     </property>
     <addaction name="actionPython_4"/>
     <addaction name="actionOpenCv_4"/>
    </widget>
    <addaction name="menurgb_2_gray"/>
    <addaction name="menu_2"/>
    <addaction name="menuesikleme"/>
    <addaction name="menunegatif"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>_2_</string>
    </property>
    <widget class="QMenu" name="menualcak_geciren_filtre">
     <property name="title">
      <string>alcak geciren filtre</string>
     </property>
     <addaction name="actionPython_5"/>
     <addaction name="actionOpenCv_5"/>
    </widget>
    <widget class="QMenu" name="menugauss_filtre">
     <property name="title">
      <string>gauss filtre</string>
     </property>
     <addaction name="actionPython_6"/>
     <addaction name="actionOpenCv_6"/>
    </widget>
    <widget class="QMenu" name="menumean_filtre">
     <property name="title">
      <string>mean filtre</string>
     </property>
     <addaction name="actionPython_7"/>
     <addaction name="actionOpenCv_7"/>
    </widget>
    <widget class="QMenu" name="menumedyan_filtre">
     <property name="title">
      <string>medyan filtre</string>
     </property>
     <addaction name="actionPython_8"/>
     <addaction name="actionOpenCv_8"/>
    </widget>
    <addaction name="menualcak_geciren_filtre"/>
    <addaction name="menugauss_filtre"/>
    <addaction name="menumean_filtre"/>
    <addaction name="menumedyan_filtre"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>_3_</string>
    </property>
    <widget class="QMenu" name="menulaplas_filtresi">
     <property name="title">
      <string>laplas filtresi</string>
     </property>
     <addaction name="actionPython_9"/>
     <addaction name="actionOpenCv_9"/>
    </widget>
    <widget class="QMenu" name="menusobel_filtresi">
     <property name="title">
      <string>sobel filtresi</string>
     </property>
     <addaction name="actionPython_10"/>
     <addaction name="actionOpenCv_10"/>
    </widget>
    <widget class="QMenu" name="menuprewit_filtresi">
     <property name="title">
      <string>prewit filtresi</string>
     </property>
     <addaction name="actionPython_11"/>
     <addaction name="actionOpenCv_11"/>
    </widget>
    <addaction name="menulaplas_filtresi"/>
    <addaction name="menusobel_filtresi"/>
    <addaction name="menuprewit_filtresi"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>_4_</string>
    </property>
    <widget class="QMenu" name="menuhistogram_esitleme">
     <property name="title">
      <string>histogram esitleme</string>
     </property>
     <addaction name="actionPython_13"/>
     <addaction name="actionOpenCv_13"/>
    </widget>
    <widget class="QMenu" name="menukontrast_germe">
     <property name="title">
      <string>kontrast germe</string>
     </property>
     <addaction name="actionPython_12"/>
     <addaction name="actionOpenCv_12"/>
    </widget>
    <addaction name="menukontrast_germe"/>
    <addaction name="menuhistogram_esitleme"/>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="title">
     <string>_5_</string>
    </property>
    <widget class="QMenu" name="menuyayma">
     <property name="title">
      <string>yayma</string>
     </property>
     <addaction name="actionPython_14"/>
     <addaction name="actionOpenCv_14"/>
    </widget>
    <widget class="QMenu" name="menuasindirma">
     <property name="title">
      <string>asindirma</string>
     </property>
     <addaction name="actionPython_15"/>
     <addaction name="actionOpenCv_15"/>
    </widget>
    <widget class="QMenu" name="menuopen">
     <property name="title">
      <string>open</string>
     </property>
     <addaction name="actionPython_16"/>
     <addaction name="actionOpenCv_16"/>
    </widget>
    <widget class="QMenu" name="menuclose">
     <property name="title">
      <string>close</string>
     </property>
     <addaction name="actionPython_17"/>
     <addaction name="actionOpenCv_17"/>
    </widget>
    <addaction name="menuyayma"/>
    <addaction name="menuasindirma"/>
    <addaction name="menuopen"/>
    <addaction name="menuclose"/>
   </widget>
   <widget class="QMenu" name="menu_8">
    <property name="title">
     <string>_6_</string>
    </property>
    <widget class="QMenu" name="menunetlestir">
     <property name="title">
      <string>netlestir</string>
     </property>
     <addaction name="actionPython_18"/>
     <addaction name="actionOpenCv_18"/>
    </widget>
    <addaction name="menunetlestir"/>
   </widget>
   <widget class="QMenu" name="menu_9">
    <property name="title">
     <string>_7_</string>
    </property>
    <widget class="QMenu" name="menukonvolusyon">
     <property name="title">
      <string>konvolusyon</string>
     </property>
     <addaction name="actionPython_19"/>
     <addaction name="actionOpenCv_19"/>
    </widget>
    <addaction name="menukonvolusyon"/>
   </widget>
   <widget class="QMenu" name="menu_10">
    <property name="title">
     <string>_8_</string>
    </property>
    <widget class="QMenu" name="menucevir">
     <property name="title">
      <string>cevir</string>
     </property>
     <addaction name="actionPython_20"/>
     <addaction name="actionOpenCv_20"/>
    </widget>
    <widget class="QMenu" name="menuboyutland_r">
     <property name="title">
      <string>boyutlandır</string>
     </property>
     <addaction name="actionPython_21"/>
     <addaction name="actionOpenCv_21"/>
    </widget>
    <addaction name="menucevir"/>
    <addaction name="menuboyutland_r"/>
   </widget>
   <addaction name="menuDosya"/>
   <addaction name="menu_1"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_7"/>
   <addaction name="menu_8"/>
   <addaction name="menu_9"/>
   <addaction name="menu_10"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionResim_A">
   <property name="text">
    <string>Resim Aç</string>
   </property>
  </action>
  <action name="actionPython">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionOpenCv">
   <property name="text">
    <string>OpenCv</string>
   </property>
  </action>
  <action name="actionPython_2">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionOpenCv_2">
   <property name="text">
    <string>OpenCv</string>
   </property>
  </action>
  <action name="actionPython_3">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionOpenCv_3">
   <property name="text">
    <string>OpenCv</string>
   </property>
  </action>
  <action name="actionPython_4">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionOpenCv_4">
   <property name="text">
    <string>OpenCv</string>
   </property>
  </action>
  <action name="actionPython_5">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionOpenCv_5">
   <property name="text">
    <string>OpenCv</string>
   </property>
  </action>
  <action name="actionPython_6">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionOpenCv_6">
   <property name="text">
    <string>OpenCv</string>
   </property>
  </action>
  <action name="actionPython_7">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionOpenCv_7">
   <property name="text">
    <string>OpenCv</string>
   </property>
  </action>
  <action name="actionPython_8">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionOpenCv_8">
   <property name="text">
    <string>OpenCv</string>
   </property>
  </action>
  <action name="actionkontrast_germe">
   <property name="text">
    <string>kontrast germe</string>
   </property>
  </action>
  <action name="actionhistogram_esitleme">
   <property name="text">
    <string>histogram esitleme</string>
   </property>
  </action>
  <action name="actionlaplacian_filresi">
   <property name="text">
    <string>laplacian filresi</string>
   </property>
  </action>
  <action name="actionPython_9">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionOpenCv_9">
   <property name="text">
    <string>OpenCv</string>
   </property>
  </action>
  <action name="actionPython_10">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionOpenCv_10">
   <property name="text">
    <string>OpenCv</string>
   </property>
  </action>
  <action name="actionPython_11">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionOpenCv_11">
   <property name="text">
    <string>OpenCv</string>
   </property>
  </action>
  <action name="actionPython_12">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionOpenCv_12">
   <property name="text">
    <string>OpenCv</string>
   </property>
  </action>
  <action name="actionPython_13">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionOpenCv_13">
   <property name="text">
    <string>OpenCv</string>
   </property>
  </action>
  <action name="actionPython_14">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionOpenCv_14">
   <property name="text">
    <string>OpenCv</string>
   </property>
  </action>
  <action name="actionPython_15">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionOpenCv_15">
   <property name="text">
    <string>OpenCv</string>
   </property>
  </action>
  <action name="actionPython_16">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionOpenCv_16">
   <property name="text">
    <string>OpenCv</string>
   </property>
  </action>
  <action name="actionPython_17">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionOpenCv_17">
   <property name="text">
    <string>OpenCv</string>
   </property>
  </action>
  <action name="actionPython_18">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionOpenCv_18">
   <property name="text">
    <string>OpenCv</string>
   </property>
  </action>
  <action name="actionPython_19">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionOpenCv_19">
   <property name="text">
    <string>OpenCv</string>
   </property>
  </action>
  <action name="actiongoruntu_cevir">
   <property name="text">
    <string>goruntu cevir</string>
   </property>
  </action>
  <action name="actionPython_20">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionOpenCv_20">
   <property name="text">
    <string>OpenCv</string>
   </property>
  </action>
  <action name="actionPython_21">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionOpenCv_21">
   <property name="text">
    <string>OpenCv</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
